doc/newfeat.texi

```diff
@@ -59,6 +59,13 @@ The following have been made obsolete, see their documentation for more detail:
 
 @itemize @bullet
 
+@item Libraries for literate data analysis.
```

Comment:
Did you forget to say that they are "obsoleted and not loaded by default"?

Reply:
Indeed, I'll add that.